### DISCUSSION TITLE
OTHER_DOC

### DISCUSSION THREAD

**Discussion by SandraEliz** [open-sdg/open-sdg#1867] [General]
Hello

In the documentation they indicate that with other_doc you can add links to documents.

Can you give me an example of how to implement it?

Thank you

**Comment by sarah-beasley** [comment]
Hi, Yes we can. It is a space here to add a link to something else useful, but since you are using a .yml file for your metadata, it needs to be in the correct format. This means it should look something like below - I have used the UN metadata repository as an example for the formatting. The link in the speech marks, and the words you want to appear in after.

<a href="https://unstats.un.org/sdgs/metadata/">UN Metadata Repository</a>

(I have had to add an image below of the text below, as it keeps automatically simplifying the link to as above!)

![image](https://user-images.githubusercontent.com/106670682/200839579-5f770a8a-112f-4f09-a009-3fb32904e54f.png)


Hope this makes sense!

**Comment by SandraEliz** [comment]
14 / 5.000

Thank you very much
